app/services/stripe_service/api/stripe_api_wrapper.rb

```diff
@@ -309,9 +309,11 @@ def create_payment_intent(community:, seller_account_id:, payment_method_id:, am
           currency: currency,
           confirmation_method: 'manual',
           confirm: true,
-          application_fee_amount: fee,
           on_behalf_of: seller_account_id,
-          transfer_data: {destination: seller_account_id},
+          transfer_data: {
+            destination: seller_account_id,
+            amount: amount - fee
```

Comment:
@bladealslayer This fixes amount of transfer later paid out for seller.

Reply:
@aivils true! A month ago, Stripe did not support passing amount here when creating payment intents, so I didn't know this is now possible. Looks like it works!